pygeoapi/api.py

```diff
@@ -855,8 +855,16 @@ def get_collection_item(self, headers_, format_, dataset, identifier):
         p = load_plugin('provider',
                         self.config['datasets'][dataset]['provider'])
 
-        LOGGER.debug('Fetching id {}'.format(identifier))
-        content = p.get(identifier)
+        try:
+            LOGGER.debug('Fetching id {}'.format(identifier))
+            content = p.get(identifier)
+        except ProviderQueryError:
```

Comment:
Perhaps we can log the actual error message instead?  See below:

```python
        except ProviderQueryError as err:
            exception = {
                'code': 'NoApplicableCode',
                'description': 'query error (check logs)'
            }
            LOGGER.error(err)
            return headers_, 500, json.dumps(exception)
```

Reply:
Good point!